src/sync-client.js

```diff
@@ -1289,5 +1292,6 @@ module.exports = {
   setCloudHandler: self.setCloudHandler,
   doCloudCall: self.doCloudCall,
   setStorageAdapter: self.setStorageAdapter,
-  setHashMethod: self.setHashMethod
+  setHashMethod: self.setHashMethod,
+  isOnline : self.isOnline
```

Comment:
sure, it is a new method anyway to we can definitely implement it in the proper way.